src/imitation/rewards/discrim_net.py

```diff
@@ -12,7 +12,7 @@
 from imitation.util import serialize
 
 
-class DiscrimNet(serialize.Serializable):
+class DiscrimNet(serialize.Serializable, ABC):
```

Comment:
This is unnecessary. Subclasses of an abstract class are abstract until they implement all abstract methods. See https://stackoverflow.com/questions/48814211/python3-how-does-one-define-an-abstract-subclass-from-an-existing-abstract-cla

Reply:
Oh I see, `pytype` is complaining about it, very odd.